projects/1.18/assets/farmers-delight/farmersdelight/lang/zh_cn.json

```
@@ -178,16 +188,17 @@
 
   "itemGroup.farmersdelight": "农夫乐事",
 
+  "farmersdelight.container.recipe_book.cookable": "仅显示可烹饪",
   "farmersdelight.container.cooking_pot": "厨锅",
   "farmersdelight.container.cooking_pot.served_on": "需盛装于：%s",
-  "farmersdelight.container.cooking_pot.not_heated": "需要下部热源",
+  "farmersdelight.container.cooking_pot.not_heated": "下方需要热源",
   "farmersdelight.container.cooking_pot.heated": "已受热",
   "farmersdelight.container.basket": "篮子",
   "farmersdelight.container.cabinet": "厨柜",
 
   "farmersdelight.tooltip.cooking_pot.empty": "空",
-  "farmersdelight.tooltip.cooking_pot.single_serving": "目前的1份食物：",
-  "farmersdelight.tooltip.cooking_pot.many_servings": "目前的%s份食物：",
+  "farmersdelight.tooltip.cooking_pot.single_serving": "装有1份食物：",
+  "farmersdelight.tooltip.cooking_pot.many_servings": "装有%s份食物：",

```

```suggestion
  "farmersdelight.tooltip.cooking_pot.single_serving": "装有1份：",
  "farmersdelight.tooltip.cooking_pot.many_servings": "装有%s份：",
```

原文是`Hold 1 serving of: `

---

我刚刚进游戏看了一下，你的翻译明显更合理，应该改成你的